src/main/resources/db/migration/V4_4__drop_columns_dialogmeldingopplysninger.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE DIALOGMELDINGOPPLYSNINGER
+DROP COLUMN pasient_aktoer_id,
+DROP COLUMN lege_aktoer_id,
+DROP COLUMN tss_id;
```

Comment:
Ser ikke noe behov for å ha disse feltene persistert i padm2.

Reply:
tss_id, kan være lurt å ha. Da når appen ble laget så ble den brukt til å "identifisere", hvilken rolle og sted behandler sendte dialogmeldingen fra. Dei 2 andre feltene kan nok fjernes

Reply:
Vi henter den fram fortsatt og sender den med til Arena, men ser ikke noe behov for å persistere den. Dessuten trenger vi da heller ikke gjøre noe kall til noen ekstern app før vi lagrer (og reduserer dermed sannsynligheten for at meldinger havner på backout-køen).